### PR TITLE
[DEV][Unit test]Add unit test for otx.algorithms.action.adapters.openvino

### PR DESCRIPTION
**Pull request by jaegukhyun**
This PR includes adding unit test codes for openvino adapters in action task

**COVERAGE**
image

### CODE REVIEW

File: tests/unit/algorithms/action/adapters/openvino/test_action_dataloader.py [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+
+    dataset = DatasetEntity(items)
+    assert _is_multi_video(dataset) is False
+
+    items.append(MockDatasetItemEntity(metadata=[MetadataItemEntity(data=VideoMetadata("1", 0, False))]))
+    items.append(MockDatasetItemEntity(metadata=[MetadataItemEntity(data=VideoMetadata("1", 1, False))]))
+    items.append(MockDatasetItemEntity(metadata=[MetadataItemEntity(data=VideoMetadata("1", 2, False))]))
+
+    dataset = DatasetEntity(items)
+    assert _is_multi_video(dataset) is True
```
Comment (eunwoosh) [inline tests/unit/algorithms/action/adapters/openvino/test_action_dataloader.py:133]:
```suggestion
    assert _is_multi_video(dataset)
```
As far as I know, `is True` make an error during pre-commit.

File: tests/unit/algorithms/action/adapters/openvino/test_action_dataloader.py [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+        annotation_scene: Optional[AnnotationSceneEntity] = None,
+    ):
+        self.__metadata = metadata
+        self.__annotation_scene: AnnotationSceneEntity = annotation_scene
+
+    def get_metadata(self) -> List[MetadataItemEntity]:
+        return self.__metadata
+
+
+class MockImage(Image):
```
Comment (eunwoosh) [inline tests/unit/algorithms/action/adapters/openvino/test_action_dataloader.py]: This is a just question. Why don't you use `MockImage` class in tests/unit/algorithms/action/test_helpers.py but instead re-define it?

File: tests/unit/algorithms/action/adapters/openvino/test_action_dataloader.py (hunk 1 above)
Comment (jaegukhyun) [reply tests/unit/algorithms/action/adapters/openvino/test_action_dataloader.py:133]: Really? I think == True raise error

Comment (eunwoosh) [reply tests/unit/algorithms/action/adapters/openvino/test_action_dataloader.py:133]: Sorry. That's my mistake.

File: tests/unit/algorithms/action/adapters/openvino/test_action_dataloader.py (hunk 2 above)
Comment (jaegukhyun) [reply tests/unit/algorithms/action/adapters/openvino/test_action_dataloader.py]: I didn't expect previous PR is merged before I create this PR.
Thanks for updating me. I change codes to remove some duplicated implementations

**Review by eunwoosh — APPROVED** [review]

**Review by goodsong81 — APPROVED** [review]

**Pull request merged**